src/org/usfirst/frc/team166/robot/RobotMap.java

```diff
@@ -1,18 +1,35 @@
 package org.usfirst.frc.team166.robot;
+
 /**
- * The RobotMap is a mapping from the ports sensors and actuators are wired into
- * to a variable name. This provides flexibility changing wiring, makes checking
- * the wiring easier and significantly reduces the number of magic numbers
+ * The RobotMap is a mapping from the ports sensors and actuators are wired into to a variable name. This provides
+ * flexibility changing wiring, makes checking the wiring easier and significantly reduces the number of magic numbers
  * floating around.
  */
 public class RobotMap {
-    // For example to map the left and right motors, you could define the
-    // following variables to use with your drivetrain subsystem.
-    // public static int leftMotor = 1;
-    // public static int rightMotor = 2;
-    
-    // If you are using multiple modules, make sure to define both the port
-    // number and the module. For example you with a rangefinder:
-    // public static int rangefinderPort = 1;
-    // public static int rangefinderModule = 1;
+
+	// Joysticks
+	public static final int Copilot = 2;
```

Comment:
Assigning numbers to people is rude, even if it IS just your copilot.

(What is the variable? `CopilotJoystickNumber`?)